src/main/java/com/employeesService/EmployeesService/model/DepartmentEmployeeDTO.java

```diff
@@ -0,0 +1,33 @@
+package com.employeesService.EmployeesService.model;
+
+import javax.persistence.JoinColumn;
+import java.io.Serializable;
+import java.time.LocalDate;
+
+public class DepartmentEmployeeDTO implements Serializable {
+
+    private LocalDate fromDate;
+    private LocalDate toDate;
+
+    @JoinColumn(name = "emp_no")
```

Comment:
Are you sure this is needed in the DTO? `@JoinColumn` is from javax.persistence package, it does not have to do anything with the DTOs which are going to be serialized to JSON